tests/integration/sentinel/transitions/registration.spec.js

```diff
@@ -98,6 +98,150 @@ describe('registration', () => {
+          type: 'person',
+          created_by: 'person',
+          source_id: patientNameAndPhone._id,
+        });
+      });
+  });
+
+  it('should not create patient from report doc when provided invalid phone', () => {
+    const patient_phone = '+97796666';
+    const patient_id =uuid();
+
+    const patientNameAndInvalidPhone = { // has just the `patient_name` field, and should create this person
+      _id: patient_id,
+      type: 'data_record',
+      form: 'FORM-A',
+      from: '+9779841212345',
+      fields: {
+        patient_name: 'Minerva',
+        phone_number: patient_phone
+      },
+      reported_date: moment().valueOf(),
+      contact: {
+        _id: 'person',
+        parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } }
+      }
+    };
+
+    const docs = [
+      patientNameAndInvalidPhone
+    ];
+    const docIds = getIds(docs);
+
+    return utils
+      .updateSettings(testForm.forms.NP, 'sentinel')
+      .then(utils.saveDocs(docs))
+      .then(() => sentinelUtils.waitForSentinel(docIds))
+      .then(() => sentinelUtils.getInfoDocs(docIds))
+      .then(infos => {
+        infos.forEach(info => {
+          chai.expect(info).to.deep.nested.not.include({ 'transitions.registration.ok': false });
+        });
+      }).then(() => utils.getDocs(docIds))
+      .then(updated => {
+        chai.expect(updated[0].fields.phone_number).to.equal(patient_phone);
+        chai.expect(updated[0].patient_id).to.not.be.null;
+        const newPatientId = updated[0].patient_id;
+        return getContactsByReference([newPatientId, 'venus']);
+      })
+      .then(patients => {
+        chai.expect(patients[0]).to.be.undefined;
+      });
+  });
+
+  it('should fail transition on invalid phone', () => {
+    const patient_phone = '+97796666';
+    const patientNameAndInvalidPhone = { // has just the `patient_name` field, and should create this person
+      _id: uuid(),
+      type: 'data_record',
+      form: 'FORM-A',
+      from: '+9779841212345',
+      fields: {
+        patient_name: 'Minerva',
+        phone_number: patient_phone
+      },
+      reported_date: moment().valueOf(),
+      contact: {
+        _id: 'person',
+        parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } }
+      }
+    };
+
+    const docs = [
+      patientNameAndInvalidPhone
+    ];
+    const docIds = getIds(docs);
+
+    return utils
+      .updateSettings(testForm.forms.NP, 'sentinel')
+      .then(utils.saveDocs(docs))
+      .then(() => sentinelUtils.waitForSentinel(docIds))
+      .then(() => sentinelUtils.getInfoDocs(docIds))
+      .then(infos => {
+        infos.forEach(info => {
+          chai.expect(info).to.deep.nested.not.include({ 'transitions.registration.ok': false });
+        });
+      })
+      .then(() => utils.getDocs(docIds))
+      .then(updated => {
+        chai.expect(updated.patient_id).to.be.undefined;
```

Comment:
I think that `updated` will be an array here. 
```suggestion
chai.expect(updated[0].patient_id).to.be.undefined;
```

Reply:
Changed thanks.